app/managers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+from dataclasses import dataclass
+from app.models import Actor
```

Comment:
fix imports ordering according to PEP8 https://peps.python.org/pep-0008/#imports